integration_test.go

```diff
@@ -3,59 +3,80 @@ package exasol_test
 import (
 	"context"
 	"database/sql"
-	_ "github.com/exasol/exasol-driver-go"
-	"github.com/stretchr/testify/suite"
-	"github.com/testcontainers/testcontainers-go"
-	"github.com/testcontainers/testcontainers-go/wait"
+	"fmt"
 	"log"
 	"strings"
 	"testing"
 	"time"
+
+	"github.com/exasol/exasol-driver-go"
+
+	"github.com/stretchr/testify/suite"
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
 )
 
 type IntegrationTestSuite struct {
 	suite.Suite
 	ctx             context.Context
 	exasolContainer testcontainers.Container
-	port            string
+	port            int
 }
 
 func TestIntegrationSuite(t *testing.T) {
 	suite.Run(t, new(IntegrationTestSuite))
 }
 
 func (suite *IntegrationTestSuite) SetupSuite() {
+	if testing.Short() {
+		return
+	}
 	suite.ctx = getContext()
 	suite.exasolContainer = runExasolContainer(suite.ctx)
 	suite.port = getExasolPort(suite.exasolContainer, suite.ctx)
 }
 
 func (suite *IntegrationTestSuite) TestConnect() {
-	exasol, _ := sql.Open("exasol", "exa:localhost:"+suite.port+";user=sys;password=exasol;encryption=0")
-	rows, _ := exasol.Query("SELECT 2 FROM DUAL")
+	if testing.Short() {
+		suite.T().Skip("skipping integration test")
+	}
+	db, _ := sql.Open("exasol", fmt.Sprintf("exa:localhost:%d;user=sys;password=exasol;encryption=0", suite.port))
+	rows, _ := db.Query("SELECT 2 FROM DUAL")
 	columns, _ := rows.Columns()
 	suite.Equal("2", columns[0])
 }
 
 func (suite *IntegrationTestSuite) TestConnectWithWrongPort() {
-	exasol, _ := sql.Open("exasol", "exa:localhost:1234;user=sys;password=exasol;encryption=0")
+	if testing.Short() {
+		suite.T().Skip("skipping integration test")
+	}
```

Comment:
Have you tested if we really need this in each method when we place it once in `TestIntegrationSuite`? I thought it was a main entry point into the test